src/components/select/select.component.ts

```diff
@@ -8,6 +8,7 @@ import { debounceTime, delay, distinctUntilChanged, filter, map, take, takeUntil
 import { InfiniteScrollLoadFunction } from '../../directives/infinite-scroll/index';
 import { TagInputComponent } from '../tag-input/index';
 import { TypeaheadComponent, TypeaheadKeyService, TypeaheadOptionEvent } from '../typeahead/index';
+import { TagApi } from '../../../dist/components/tag-input';

```

Comment:
We can't import anything from dist because it either won't exist at compile time or will be out of date. Try adding this to the above import from '../tag-input/index'.

Reply:
That is it changed now. I had at this one but I must have overridden it.